l10n_it_fatturapa_in/models/company.py

```diff
@@ -12,38 +13,45 @@ class ResCompany(models.Model):
     sconto_maggiorazione_product_id = fields.Many2one(
         'product.product', 'Discount Supplement Product',
         help="Product used to model ScontoMaggiorazione XML element on bills."
-        )
+    )
+
+    arrotondamenti_attivi_account_id = fields.Many2one(
+        'account.account', 'Rounding Up Account',
+        domain=[('deprecated', '=', False)],
+        help="Account used to rounding up amount on bills."
+    )
+    arrotondamenti_passivi_account_id = fields.Many2one(
+        'account.account', 'Rounding Down Account',
+        domain=[('deprecated', '=', False)],
+        help="Account used to rounding down amount on bills."
+    )
+    arrotondamenti_tax_id = fields.Many2one(
+        'account.tax', 'Rounding Tax',
+        domain=[('type_tax_use', '=', 'purchase'), ('amount', '=', 0.0)],
+        help="Tax used to both up and down roundings on bills."
```

Comment:
```suggestion
        help="Tax used to both round up and down bills amount."
```
oppure `amount on bills`, l'importante è che ci sia uniformità con le stringhe simili alle righe 21 e 26